hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+
```

Comment:
Please add a copyright header here:

```suggestion
© 2020 Red Hat, Inc. and others.
```

Reply:
`controller-gen` has a `year` argument I can use it to automate the process